web/src/components/pages.tsx

```diff
@@ -379,35 +382,53 @@ class Pages extends React.Component<PagesProps, PagesState> {
                 <div className="divider" />
                 <a href="javascript:void(0)" onClick={this.toggleContactModal}>
                   <ContactIcon />
-                  <div>Contact</div>
+                  <Localized id="contact">
+                    <div />
+                  </Localized>
                 </a>
               </div>
               <div id="moz-links">
                 <div className="content">
                   <Logo reverse={true} />
                   <div className="links">
                     <p>
-                      <Link to={URLS.PRIVACY}>Privacy</Link>
-                      <Link to={URLS.TERMS}>Terms</Link>
-                      <a
-                        target="_blank"
-                        href="https://www.mozilla.org/en-US/privacy/websites/#cookies">
-                        Cookies
-                      </a>
-                      <Link to={URLS.FAQ}>FAQ</Link>
+                      <Localized id="privacy">
+                        <Link to={URLS.PRIVACY} />
+                      </Localized>
+                      <Localized id="terms">
+                        <Link to={URLS.TERMS} />
+                      </Localized>
+                      <Localized id="cookies">
+                        <a
+                          target="_blank"
+                          href="https://www.mozilla.org/en-US/privacy/websites/#cookies"
+                        />
+                      </Localized>
+                      <Localized id="faq">
+                        <Link to={URLS.FAQ}>FAQ</Link>
+                      </Localized>
                     </p>
                     <p>
-                      Content available under a&nbsp;<a
-                        target="_blank"
-                        rel="noopener noreferrer"
-                        href="https://www.mozilla.org/en-US/foundation/licensing/website-content/">
-                        Creative Commons license
-                      </a>
+                      <Localized
+                        id="content-license-text"
+                        $link={
+                          <Localized id="content-license">
```

Comment:
If you can wait a little bit for the new version of `fluent-react`, this pattern will become obsolete. Instead, you'll be able to just use markup in the translation:

```properties
content-license-text = Content available under a <a>Creative Commons license</a>.
```

```jsx
<Localized
  id="content-license-text"
  a={<a target="…" rel="…" href="…"/>}
>
  <span />
</Localized>
```